pkg/apis/garden/v1beta1/types.go

```diff
@@ -889,10 +889,10 @@ type Kube2IAMRole struct {
 	Policy string `json:"policy"`
 }
 
-// Backup holds information about the backup interval and maximum.
+// Backup holds information about the backup schedule and maximum.
 type Backup struct {
-	// IntervalInSecond defines the interval in seconds how often a backup is taken from etcd.
-	IntervalInSecond int `json:"intervalInSecond"`
+	// Schedule defines the cron schedule according to which a backup is taken from etcd.
+	Schedule string `json:"schedule"`
 	// Maximum indicates how many backups should be kept at maximum.
 	Maximum int `json:"maximum"`
```

Comment:
If they are optional then they should be pointers with `// +optional` tag and the `omitempty` struct tag.